apps.json

```diff
@@ -329,14 +329,14 @@
     "screenshots": ["https://i.imgur.com/ehRwmIg.png", "https://i.imgur.com/oudVL8I.png"]
   },
   {
-    "name": "Purple",
-    "slug": "purple",
+    "name": "Jam",
+    "slug": "jam",
     "description": "Farcaster Superpowers: AI-powered search, live feeds, trending topics, suggested people, and more!",
     "categories": ["discovery", "search", "notifications"],
-    "url": "https://getpurple.xyz/",
+    "url": "https://jam.so/",
     "founders_username": ["betashop"],
-    "logo": "https://i.imgur.com/xGgovOZ.png",
-    "screenshots": ["https://i.imgur.com/AqupRT2.png"]
+    "logo": "https://i.imgur.com/a/S1NyQ35.png",
+    "screenshots": ["https://i.imgur.com/a/211EgrW.png"]
```

Comment:
@shnoodles urls to the images should be the direct image url like https://i.imgur.com/xffbAor.png for https://imgur.com/a/211EgrW

Do you mind replacing both the logo and screenshots? Thank you